modules/tinymce/src/core/main/ts/fmt/MatchFormat.ts

```diff
@@ -92,7 +92,11 @@ const matchItems = (dom: DOMUtils, node: Node, format, itemName: string, similar
             return;
           }
 
-          if ((!similar || format.exact) && !isEq(value, FormatUtils.normalizeStyleValue(dom, FormatUtils.replaceVars(items[key], vars), key))) {
+          const normalizedStyle = FormatUtils.normalizeStyleValue(dom, FormatUtils.replaceVars(items[key], vars), key);
+          // TINY-7227 multiple classes on same element
+          const matchAttribute = Arr.exists(value.split(' '), Fun.curry(isEq, normalizedStyle));
+
+          if ((!similar || format.exact) && !matchAttribute) {
```

Comment:
Should this also check if the whole value equals what we're looking for, to preserve the current behavior, in case that it could support values with spaces in them? I don't know where exactly MatchFormat might be used, so if this is never the case, feel free to ignore this.

Reply:
Is there some attribute where spaces are not to be interpreted as a separator? Perhaps in some custom data-attribute? In that case yeah we should preserve the current behavior.

Reply:
updated, also the whole value